swagger/communication/communication.json

```diff
@@ -573,17 +573,6 @@
     }
   },
   "paths": {
-    "/metrics": {
-      "get": {
-        "operationId": "PrometheusController_index",
-        "parameters": [],
-        "responses": {
-          "200": {
-            "description": ""
-          }
-        }
-      }
-    },
     "/v3/communication/voice_call_settings": {
       "post": {
         "operationId": "SettingsController_create",
```

Comment:
Create a VoiceCallSetting
Get all VoiceCallSettings
Update VoiceCallSetting

Get all VoiceCalls (remove the staff)
Delete a VoiceCalls (not delete voice call recording by uid)
Create a VoiceCall
Retrieve a VoiceCall (not Get voice call by uid)
Update a VoiceCall  (not Update voice call by uid)

Retrieve a VoiceCallQuota (not Get quota)

Reply:
Nice to have - order the endpoints: GetList, Create, Retrieve, Update, Delete